Config.psm1

```diff
@@ -53,6 +53,8 @@ function Get-availableConfigOptionOptions {
         @{"Name" = "zip_password";
```

Comment:
Added option to generate golden images*
Note: This image is not ready to be used in a cloud. This image can be used as a starting point to generate a fully working image.